# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/WorkerPrivate.h
# Commit: 26d3176a00ef
# Full Hash: 26d3176a00ef338cef46f4ad6f8bdfb91f2f586c
# Author: Jens Stutte <jstutte@mozilla.com>
# Date: 2021-12-08 03:42:13
# Regressor Bug: 1744025
# File Overlap Count: 3
# Description:
#   Bug 1744025: Use CheckedUnsafePtr<WorkerPrivate> in WorkerGlobalScopeBase. r=dom-worker-reviewers,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D132709
# ==============================================================================

diff -r 2c8941fdcee5 -r 26d3176a00ef dom/workers/WorkerPrivate.h
--- a/dom/workers/WorkerPrivate.h	Tue Dec 07 16:47:14 2021 +0000
+++ b/dom/workers/WorkerPrivate.h	Tue Dec 07 16:47:15 2021 +0000
@@ -32,7 +32,6 @@
 #include "mozilla/dom/Worker.h"
 #include "mozilla/dom/WorkerCommon.h"
 #include "mozilla/dom/WorkerLoadInfo.h"
-#include "mozilla/dom/WorkerScope.h"
 #include "mozilla/dom/WorkerStatus.h"
 #include "mozilla/dom/workerinternals/JSSettings.h"
 #include "mozilla/dom/workerinternals/Queue.h"